.travis.yml

```diff
@@ -7,8 +7,8 @@ git:
 
 env:
   global:
-    - BYOND_MAJOR="510"
-    - BYOND_MINOR="1346"
+    - BYOND_MAJOR="511"
+    - BYOND_MINOR="1379"
```

Comment:
Will want this to be 1380, though, since that's the minor version for the new stable release.